src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <SkipWorkloadsTestingTargetsImport>false</SkipWorkloadsTestingTargetsImport>
     <InstallWorkloadForTesting>true</InstallWorkloadForTesting>
     <InstallChromeForTests Condition="'$(InstallChromeForTests)' == '' and ('$(ContinuousIntegrationBuild)' == 'true' or Exists('/.dockerenv'))">true</InstallChromeForTests>
+    <InstallV8ForTests Condition="'$(InstallV8ForTests)' == '' and ('$(ContinuousIntegrationBuild)' == 'true' or Exists('/.dockerenv'))">true</InstallV8ForTests>
```

Comment:
could add a comment explaining dockerenv here too, not important though

Reply:
I was trying to be lazy lol. It is used in a bunch of places, i'll add there too :D